### PR TITLE
Feat/condo/doma 7063/billing account fields

### PR DESCRIPTION
**Pull request by AleX83Xpert**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]
You need to extend timeout in migration, as there are lots of BillingAccounts already

**Comment by AleX83Xpert** [comment]
> You need to extend timeout in migration, as there are lots of BillingAccounts already

Done

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/billing/schema/BillingAccount.js [hunk 1]
```diff
@@ -65,6 +66,21 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             isRequired: false,
         },
 
+        isClosed: {
+            schemaDoc: 'Shows whether the billing account closed or not',
```
Comment (pahaz) [inline apps/condo/domains/billing/schema/BillingAccount.js:70]: o_O, why we just not mark it as deleted? It looks as extra complexity and we need an architecture decision and slide before

File: apps/condo/domains/billing/schema/BillingAccount.js [hunk 2]
```diff
@@ -65,6 +66,21 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
...
+            type: 'Checkbox',
+            defaultValue: false,
+            kmigratorOptions: { default: false },
+        },
+
+        ownerType: {
```
Comment (pahaz) [inline apps/condo/domains/billing/schema/BillingAccount.js:76]: may be just `type` ?

File: apps/condo/domains/billing/schema/BillingAccount.js [hunk 3]
```diff
@@ -65,6 +66,21 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             isRequired: false,
         },
 
+        isClosed: {
```
Comment (pahaz) [inline apps/condo/domains/billing/schema/BillingAccount.js:69]: all integrations should support this field? And you need to describe it logic

File: apps/condo/domains/billing/schema/BillingAccount.js (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/domains/billing/schema/BillingAccount.js:76]: `type` of account? Or `type` of billing?
`ownerType` is clear.

File: apps/condo/domains/billing/schema/BillingAccount.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/billing/schema/BillingAccount.js:70]: Schemadoc improved

**Pull request merged**